### PR TITLE
Update Get by MMH id to only send through Active alerts

### PR DESCRIPTION
**Pull request by humulla**
Update Get by MMH id to only send through Active alerts as described in [this document](https://docs.google.com/document/d/1Nb6lax6yWxnACqPx9TQ3gfdGdBK2mlkLXGeZewB8JTg/edit?usp=sharing)

### CODE REVIEW

File: CautionaryAlertsApi.Tests/DatabaseTests.cs [hunk 1]
```diff
@@ -14,6 +15,8 @@ public class DatabaseTests
         [SetUp]
         public void RunBeforeAnyTests()
         {
+            AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```
Comment (Duslerke) [inline CautionaryAlertsApi.Tests/DatabaseTests.cs:18]: This doesn't belong in the PR based on the PR title & current description.
As such, if it's intended to be in here, it would be good to at least mention it in the PR description under "notes". _(So it would be cleat that it's not a left over of some other piece of work that sneaked in without getting noticed)_

File: CautionaryAlertsApi.Tests/V1/E2ETests/GetAlertsByPersonIdE2ETests.cs [hunk 2]
```diff
@@ -57,7 +61,7 @@ public async Task GetAlertsByPersonIdReturnsAlertsIfTheyExist()
             var returnedAlerts = JsonConvert.DeserializeObject<CautionaryAlertsMMHPersonResponse>(data);
 
             returnedAlerts.PersonId.Should().Be(id);
-            returnedAlerts.Alerts.Should().HaveSameCount(alerts);
+            returnedAlerts.Alerts.Count.Should().Be(2);
```
Comment (Duslerke) [inline CautionaryAlertsApi.Tests/V1/E2ETests/GetAlertsByPersonIdE2ETests.cs:64]: It's good that we check for count, but are we sure that we're returning the correct ones?

It's possible that some code edit in the future may introduce a bug that causes the API to return alerts like so:
``` cs
 [
      { ..., IsActive: false, ... },
      { ..., IsActive: true, ... }
 ]
```
In such event, this test wouldn't be able to tell the difference between it returning only `IsActive` `true`'s and returning the above, which is a mixture.

File: CautionaryAlertsApi.Tests/V1/Gateways/UhGatewayTests.cs [hunk 3]
```diff
@@ -428,16 +432,18 @@ public async Task GetGetCautionaryAlertsByPersonIdReturnsMany()
...
+            results.First().IsActive = false;
             await TestDataHelper.SavePropertyAlertsToDb(UhContext, results).ConfigureAwait(false);
 
             // Act
             var result = await _classUnderTest.GetCautionaryAlertsByMMHPersonId(personId).ConfigureAwait(false);
 
             // Assert
             result.Should().NotBeNull();
-            result.Should().HaveSameCount(results);
+            result.Should().HaveCount(2);
```
Comment (Duslerke) [inline CautionaryAlertsApi.Tests/V1/Gateways/UhGatewayTests.cs:446]: Same as the above.

**Review by adam-tracy — APPROVED** [review]

**Comment by Duslerke** [comment]
I already asked this on the other PR, but are we sure that the infrastructure model we need to be dealing with is the `PropertyAlertNew` and not `PropertyAlert`?

**Comment by humulla** [comment]
> I already asked this on the other PR, but are we sure that the infrastructure model we need to be dealing with is the `PropertyAlertNew` and not `PropertyAlert`?

Yep it's definitely PropertAlertNew

File: CautionaryAlertsApi.Tests/V1/E2ETests/GetAlertsByPersonIdE2ETests.cs (hunk 2 above)
Comment (humulla) [reply CautionaryAlertsApi.Tests/V1/E2ETests/GetAlertsByPersonIdE2ETests.cs]: Updated the fluent assert to find results where isActive is true and then check the count
